packages/components/src/panel/actions.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	TextControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import styles from './actions.scss';
+
+function PanelActions( { actions } ) {
+	return (
+		<View style={ styles.panelActionsContainer }>
+			{ actions.map( ( { label, onPress, labelStyle } ) => {
+				return (
+					<TextControl
+						label={ sprintf( __( '%s' ), label ) }
+						separatorType="topFullWidth"
+						onPress={ onPress }
+						labelStyle={ labelStyle }
```

Comment:
I think we can apply a default style here but let it be overridden by action.labelStyle like 
labelStyle={ [defaultStyle, labelStyle] }

because as far as I know current usages all have red color

Reply:
Hmm, I was wondering how many usages (colors) we have actually. If the main and used in the most case is red one it makes sense!

Reply:
actually we don't have any other colors for this currently